bot/resources/easter/easter_egg_facts.json

```diff
@@ -0,0 +1,22 @@
+    "Members of the Greek Orthodox faith often paint their Easter eggs red, which symbolizes Jesus\" blood and his victory over death. The color red, symbolizes renewal of life, such as, Jesus\" resurrection.",
+    "Eggs rolling take place in many parts of the world which symbolizes stone which was rolled away from the tomb where Jesus\" body was laid after his death.",
+    "Easter eggs have been considered as a symbol of fertility, rebirth and new life. The custom of giving eggs has been derived from Egyptians, Persians, Gauls, Greeks and Romans.",
+    "The first chocolate Easter egg was made by Fry\"s in 1873. Before this, people would give hollow cardboard eggs, filled with gifts.",
+    "The tallest chocolate Easter egg was made in Italy in 2011. Standing 10.39 metres tall and weighing 7,200 kg, it was taller than a giraffe and heavier than an elephant.",
+    "The largest ever Easter egg hunt was in Florida, where 9,753 children searched for 501,000 eggs.",
+    "In 2007, an Easter egg covered in diamonds sold for almost £9 million. Every hour, a cockerel made of jewels pops up from the top of the Faberge egg, flaps its wings four times, nods its head three times and makes a crowing noise. The gold-and-pink enamel egg was made by the Russian royal family as an engagement gift for French aristocrat Baron Edouard de Rothschild.",
+    "The White House held their first official egg roll in 1878 when Rutherford B. Hayes was the President. It is a race in which children push decorated, hard-boiled eggs across the White House lawn as an annual event held the Monday after Easter. In 2009, the Obamas hosted their first Easter egg roll with the theme, ‘Let\"s go play\" which was meant to encourage young people to lead healthy and active lives.",
+    "80 million chocolate Easter eggs are sold each year.  This accounts for 10% of Britain\"s annual spending on chocolate!",
+    "The tradition of giving eggs at Easter has been traced back to Egyptians, Persians, Gauls, Greeks and Romans, who saw the egg as a symbol of life.",
+    "Medieval Easter eggs were boiled with onions to give them a golden sheen.  Edward I, however, went one better and in 1290 he ordered 450 eggs to be covered in gold leaf and given as Easter gifts.",
+    "The first chocolate Easter egg was produced in 1873 by Fry\"s.  Before this, people would give hollow cardboard eggs, filled with gifts.",
+    "John Cadbury soon followed suit and made his first Cadbury Easter egg in 1875.  By 1892 the company was producing 19 different lines, all made from dark chocolate."
+  ]
+}
```

Comment:
This could probably just be a top-level array:
```json
[
    "The first story of a rabbit (later named the \"Easter Bunny\") hiding eggs in a garden was published in 1680",
    "Rabbits are known to be prolific pro creators and are an ancient symbol of fertility and new life. The German immigrants brought the tale of Easter Bunny in the 1700s with the tradition of an egg-laying hare called ‘Osterhase\". The kids then would make nests in which the creature would lay coloured eggs. The tradition has been revolutionized in the form of candies and gifts instead of eggs.",
  # ...
```